state/executor.go

```diff
@@ -345,6 +345,14 @@ func (e *BlockExecutor) execute(ctx context.Context, state types.State, block *t
 		}
 	})
 
+	if e.fraudProofsEnabled {
+		isr, err := e.getAppHash()
+		if err != nil {
+			return nil, err
+		}
+		ISRs = append(ISRs, isr)
+	}
```

Comment:
do you not need to increment `currentIsrIndex`?

Reply:
yes, great catch